paper-spinner-behavior.html

```diff
@@ -0,0 +1,104 @@
+<!--
+@license
+Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+Code distributed by Google as part of the polymer project is also
+subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+
+<script>
+
+  /** @polymerBehavior */
+  Polymer.PaperSpinnerBehavior = {
+
+    listeners: {
+      'animationend': 'reset',
+      'webkitAnimationEnd': 'reset'
+    },
+
+    properties: {
+      /**
+       * Displays the spinner.
+       *
+       * @attribute active
+       * @type boolean
+       * @default false
+       */
+      active: {
+        type: Boolean,
+        value: false,
+        reflectToAttribute: true,
+        observer: '_activeChanged'
+      },
+
+      /**
+       * Alternative text content for accessibility support.
+       * If alt is present, it will add an aria-label whose content matches alt when active.
+       * If alt is not present, it will default to 'loading' as the alt value.
+       *
+       * @attribute alt
+       * @type string
+       * @default 'loading'
+       */
+      alt: {
+        type: String,
+        value: 'loading',
+        observer: '_altChanged'
+      },
+
+      /**
+       * True when the spinner is going from active to inactive. This is represented by a fade
+       * to 0% opacity to the user.
+       */
+      _coolingDown: {
+        type: Boolean,
+        value: false
+      },
+
+      _spinnerContainerClassName: {
+        type: String,
+        computed: '_computeSpinnerContainerClassName(active, _coolingDown)'
+      }
+    },
+
+    _computeSpinnerContainerClassName: function(active, coolingDown) {
+      return [
+        active || coolingDown ? 'active' : '',
+        coolingDown ? 'cooldown' : ''
+      ].join(' ');
```

Comment:
We really need to create a `className` helper. I think @kevinpschaaf had something in the works for this. I pinged him about it. No action required, just observing.